src/Backend/MessageManagerBackendDispatcher.php

```diff
@@ -34,6 +34,8 @@ class MessageManagerBackendDispatcher extends QueueBackendDispatcher
     protected $default;
 
     /**
+     * NEXT_MAJOR: Remove $messageManager parameter.
+     *
      * @param MessageManagerInterface $messageManager Only used in compiler pass
```

Comment:
This was added in https://github.com/sonata-project/SonataNotificationBundle/pull/65, but the only compiler pass I see is https://github.com/sonata-project/SonataNotificationBundle/blob/8cd7978dbdc5c396cf3351d8df8c962aeabf902e/src/DependencyInjection/Compiler/NotificationCompilerPass.php and looks like it is not used there (and I don't know why it would need it);